compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/DiagnosticsCodes.java

```diff
@@ -74,6 +74,10 @@ public enum DiagnosticsCodes {
     PERSIST_123("PERSIST_123", "associated entity does not contain any primary keys: the key " +
             "should have a valid value", ERROR),
     PERSIST_124("PERSIST_124", "The entity ''{0}'' should be a closed record.", ERROR),
+    PERSIST_125("PERSIST_125", "relation annotation cannot be attached without the entity annotation",
```

Comment:
Whats the difference between 117 amd 125?

Reply:
I prefer the wording in 117

Reply:
We can't add the relation annotation if the field type is not another entity field type. 117 is created to this validation.
Eg: 
```
public type MultipleAssociations record {|
    readonly int id;
    @persist:Relation
    string name;
}
```

Reply:
Lets word this differently. IMO both messages are more or less the same and I thought it meant the same thing. We have to be clear on messaging